network-api/networkapi/wagtailpages/pagemodels/customblocks/card_grid.py

```diff
@@ -0,0 +1,41 @@
+from wagtail.core import blocks
+from wagtail.images.blocks import ImageChooserBlock
+from django.core.exceptions import ValidationError
+from django.forms.utils import ErrorList
+
+class CardGrid(blocks.StructBlock):
+    image = ImageChooserBlock()
+
+    alt_text = blocks.CharBlock(
+        required=False,
+        help_text='Alt text for card\'s image.'
+    )
+
+    title = blocks.CharBlock(
+        help_text='Heading for the card.'
+    )
+
+    body = blocks.TextBlock(
+        help_text='Body text of the card.'
+    )
+
+    link_url = blocks.CharBlock(
```

Comment:
Thanks for bringing this up Kalob! I reached out to Kristina and she mentioned that the links should have the ability to be internal or external.  So I will resolve this conversation since we are already using charblock